src/file.ts

```diff
@@ -1113,8 +1113,8 @@ class File extends ServiceObject<File> {
    *     NOTE: Byte ranges are inclusive; that is, `options.start = 0` and
    *     `options.end = 999` represent the first 1000 bytes in a file or object.
    *     NOTE: when specifying a byte range, data integrity is not available.
-   * @property {booblean} [returnCompressed] Disable auto decompression of the
-   *     received data. By default this option set to `false`.
+   * @property {booblean} [decompress] Disable auto decompression of the
```

Comment:
I think if you put `[decompress=true]`, that'll tip off the docs site that the default value is `true`. Also, I think it's just called a `boolean` 😅.

Reply:
Done! 😹